packages/timezone/src/components/timezone-picker/timezonePicker.tsx

```diff
@@ -54,6 +49,7 @@ export interface ITimezonePickerProps extends IProps {
 
     /**
      * Whether this component is non-interactive.
+     * This prop will be ignored if a custom child is provided.
```

Comment:
"ignored if `children` is provided" - to reference the react concept in question by name.